sdk/purview/purview-datamap-rest/test/public/sampleTest.spec.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+
+describe("My test", () => {
+  let recorder: Recorder;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
```

Comment:
remove this file if you have already added your own testings.

Reply:
removed